### ISSUE TITLE
Prevent possibility of email spoofing

### ISSUE THREAD

**Issue by frankchen07** [gitcoinco/web#8760]
### Circumstance

Gitcoin does not want email spoofing (users receiving emails from a sender address designed to make it look like us, but actually not from us).

### Description

https://mxtoolbox.com/SuperTool.aspx?action=spf%3agitcoin.co&run=toolpage

For the most part it seems we are a go according to the report above.

There may be one small change required on a txt record on the domain (needs updating in AWS (Route 53)).

The change is from ~all to -all, which puts us at risk of having some of the emails we send out not land, but it offers the most strict security (no one can spoof us due to more constrained rules). More information on this here: https://dmarcian.com/what-is-the-difference-between-spf-all-and-all/

Basically:

~all (tilde all) is a softfail, an IP that matches a mechanism with this qualifier will soft fail SPF, which means that the host should accept the mail, but mark it as an SPF failure.

-all (hyphen all) is a fail, the host should not accept the mail.

**Comment by frankchen07** [comment]
Do we have an idea of how much we are risking or securing by using -all vs ~all? (e.g any kind of data that tells us what % and type of emails might be at risk if we use a more constrained ruleset?)
